projects/Amlogic/packages/opengl-meson-t82x/package.mk

```diff
@@ -2,8 +2,8 @@
 # Copyright (C) 2017-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="opengl-meson-t82x"
-PKG_VERSION="915cb48"
-PKG_SHA256="9b5f65afa21250b67578c250da030a5829e69131ce91b2f167b01b1ed30be781"
+PKG_VERSION="3b21f97"
+PKG_SHA256="bb97c408a7b802592cb952c8ac117e810635366ab4447c5a335511fc279f3852"
```

Comment:
I get checksum ```f2f928b77e87d2c9d42583f346a95b5ec1bc572e0cb8e81f3bf9dfe1de0ef1e6 ``` for git 3b21f97

Reply:
We have to use full git sha in PKG_VERSION here.

Reply:
I did testing on KVIM2. I see lot of frame skips for  4k high bits testing videos in OSD debuging. Previous version runs skip free. I prefer performance for HW decoding over SW decoding improvement.